packages/circuit-ui/components/SideNavigation/SideNavigation.stories.tsx

```diff
@@ -119,6 +119,8 @@ export const baseArgs: SideNavigationProps = {
       externalLabel: 'Opens in a new tab',
     },
   ],
+  skipNavigationLabel: 'Skip Navigation',
+  skipNavigationHref: '#main-content',
```

Comment:
This element doesn't exist either. We could add the `id` to the "fake" main content `div`.